### PR TITLE
[ZT] WARP session durations in Access

### PR DESCRIPTION
**Pull request by ranbel**
pcx-8507

### CODE REVIEW

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-sessions.md [hunk 1]
```diff
@@ -28,6 +32,21 @@ You can configure a WARP session for any Allow policy. To configure a session:
...
+## Configure WARP sessions in Access
+
+You can allow users to log in to Access applications using their WARP session. WARP authentication is only supported for Access applications protected by Allow or Block policies.
+
+To configure WARP sessions for Access applications:
+
+1. In [Zero Trust](https://one.dash.cloudflare.com/), go to **Settings** > **WARP Client**.
+2. In **Device enrollment permissions**, select **Manage**.
+3. Go to the **Authentication** tab and enable **WARP authentication identity**.
+4. Under **Session duration**, choose a session timeout value. This timeout will apply to all Access applications that have WARP authentication enabled.
```
Comment (kennyj42) [inline content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-sessions.md:44]: idk if we want to add the call out that this will not affect Gateway WARP session re-auth's?

**Review by kennyj42 — APPROVED** [review]
looks good overall! A couple small comments and tweaks

File: content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-sessions.md [hunk 2]
```diff
@@ -28,6 +32,21 @@ You can configure a WARP session for any Allow policy. To configure a session:
...
+5. (Optional) To enable WARP authentication by default for all existing and new applications, select **Apply to all Access applications**. You can override this default setting on a per-application basis when you [create](/cloudflare-one/applications/configure-apps/) or modify an Access application.
+6. Select **Save**.
+
+Users can now authenticate once with WARP and have access to your Access applications for the configured period of time.
```
Comment (ajholland) [inline content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-sessions.md]: cc @kennyj42 we might want to edit the wording here - users auth once into the warp app, but then their warp session is determined based on when they do a login to an IdP WITH warp enabled. 

This is too wordy, but something like - "After authenticating to WARP, a users identity will be stored for the configured period of time. Subsequent logins to identity providers will be stored for the configured period of time without requiring re-authenticating to WARP"

Comment (ranbel) [reply content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-sessions.md]: Are we accurately describing this behavior in the intro paragraph?
> If the session has expired, the user will be prompted to re-authenticate with the identity provider (IdP) used to enroll in the WARP client. A user's WARP session duration resets to zero whenever they log in to the IdP, regardless of what triggered the authentication event.

Comment (ajholland) [reply content/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-sessions.md]: Yes 👍
